build/push

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
```

Comment:
Can you use the push script from ZFS local pv. It has some optimizations.

Reply:
Done